schema/BioOntologySchema.mcf

```diff
@@ -0,0 +1,5206 @@
+Node: dcid:psimiID
+typeOf: schema:Property
+name: "psimiID"
+description: "The identifier for Proteomics Standards Initiative Molecular Interaction."
+rangeIncludes: schema:Text
+domainIncludes:dcs:InteractionTypeEnum,dcs:InteractionDetectionMethodEnum,dcs:InteractionSourceEnum
+
```

Comment:
BioMINTSchema.mcf file mixes both schema and data MCF. The data MCF should not be checked into github, should not be checked in here. Can we separate them?

For example:


Node: dcid:AuthorScore
typeOf: ConfidenceScoreTypeEnum
name: "AuthorScore"
description: "The confidence score of a protein-protein interaction given by author."

Node: dcid:IntActMiScore
typeOf: ConfidenceScoreTypeEnum
name: "IntactMiScore"
description: "The confidence score of a protein-protein interaction given in IntAct Molecular Interaction Database."

Node: dcid:bio/RPN1_YEAST_RPN3_YEAST
typeOf: ProteinProteinInteraction
name: "RPN1_YEAST_RPN3_YEAST"
interactingProtein: dcs:bio/UniProt_RPN1_YEAST,dcs:bio/UniProt_RPN3_YEAST
interactionDetectionMethod: dcs:TandemAffinityPurification
interactionType: dcs:PhysicalAssociation
interactionSource: dcs:Mint
intActID: "EBI-6941860"
mintID: "MINT-1984371"
imexID: "IM-15332-8532"
confidenceScore: [0.76 dcs:IntactMiScore]
pubMedID: "16554755"
imexID: "IM-15332"
mintID: "MINT-5218454"